runtime/include/chpl-comm.h

```diff
@@ -82,14 +82,14 @@ extern const int chpl_heterogeneous;
 // before the request completes (after waiting on the returned handle)
 chpl_comm_nb_handle_t chpl_comm_get_nb(void* addr, c_nodeid_t node, void* raddr,
                                        size_t size, int32_t typeIndex,
-                                       int ln, c_string fn);
+                                       int ln, int32_t fn);
```

Comment:
Yeah, this is mostly just from going quickly through everything and changing it all. I can standardize on one method if you'd prefer, but it's not something I'm in any hurry to do (it's a fair amount of easy, boring, work).